apps/bundle-size/src/index.js

```diff
@@ -0,0 +1,12 @@
+// @ts-check
+
+require('yargs')
```

Comment:
**suggestion:** I know those are "standard" commonjs practices. but we can do better for the ESM era in node.

I'd suggest to do following:

```ts
const yargs = require('yargs');

const cliSetup = yargs
  .commandDir('commands')
  .option('verbose', {
    alias: 'v',
    type: 'boolean',
    description: 'Run with verbose logging',
    default: false,
  })
  .version(false).argv;

module.export = cliSetup
```

now with that we can get properly inferred types  for our commands, which will stay in sync (DRY/single source of truth)

```diff
// @filename  apps/bundle-size/src/commands/build.js
-/** * @param {{ verbose: boolean }} options 
+/** * @param {typeof import('../../index').cliSetup} options 
```

Reply:
Thanks for suggestion, I will try it

Reply:
Yeah, it works, done in 36ff8f3.

P.S. The single thing that I don't like that `yarns.Argv` include `[string]: unknown` that allows to do `options.foo` without errors 😮